### PR TITLE
Update Russian locale

### PR DESCRIPTION
**Pull request by NickKaramoff**
Following https://github.com/aonez/Keka/commit/b3a2a5e6fdf29bc3995fadcd1aa76aa514a661a8, I have localized new strings in Russian.
Additionally, I have updated my email and website addresses.

### CODE REVIEW

**Comment by aonez** [comment]
Thanks a lot for all the updates Nick!

File: Translations/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -233,21 +233,21 @@
...
+"Monthly coffee" = "Ежемесячный кофе";
+"Gift Keka a coffee each month" = "Угощать Keka кофе каждый месяц";
+"Monthly pair of coffees" = "Ежемесячная пара чашек кофе";
+"Gift Keka a couple coffees each month" = "Угощать Keka парой чашек кофе каждый месяц";
+"Monthly breakfast" = "Ежемесячный завтрак";
+"Gift Keka a breakfast each month" = "Угощать Keka завтраком каждый месяц";
+"Tip a coffee" = "Угостить кофе";
+"Tip Keka a coffee so it can stay awake" = "Угостить Keka кофе, чтобы не засыпать";
+"Tip a breakfast" = "Угостить завтраком";
+"Tip Keka a breakfast so it can start fresh" = "Угостить Keka завтраком для отличного начала дня";
```
Comment (aonez) [inline Translations/ru.lproj/Localizable.strings:247]: @NickKaramoff this one exceeds 45 characters so can't be used

File: Translations/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -233,21 +233,21 @@
...
+"Tip a lunch" = "Угостить обедом";
+"Tip Keka a lunch and feed it" = "Угостить Keka обедом, чтобы не голодать";
+"Tip a dinner" = "Угостить ужинов";
+"Tip Keka a dinner so it keeps rolling" = "Угостить Keka ужином, чтобы работа не останавливалась";
```
Comment (aonez) [inline Translations/ru.lproj/Localizable.strings:251]: @NickKaramoff this one also exceeds 45 characters and can't be used

File: Translations/ru.lproj/Localizable.strings [hunk 3]
```diff
@@ -233,21 +233,21 @@
...
+"Tip a bunch of food" = "Устроить пир";
+"Tip Keka a new bunch of food to keep rolling" = "Устроить Keka пир, чтобы работы не останавливалась";
```
Comment (aonez) [inline Translations/ru.lproj/Localizable.strings:253]: @NickKaramoff and also this one exceeds 45 characters and can't be used 😅

File: Translations/ru.lproj/Localizable.strings (hunk 1 above)
Comment (NickKaramoff) [reply Translations/ru.lproj/Localizable.strings:247]: Fixed in #967

**Pull request merged**